rasa/utils/tensorflow/models.py

```diff
@@ -666,7 +666,7 @@ def __init__(
         )
 
         # set up tf layers
-        self._tf_layers: Dict[Text : tf.keras.layers.Layer] = {}
+        self._tf_layers: Dict[Text, tf.keras.layers.Layer] = {}
```

Comment:
🤯

Reply:
it is very surprising that pytype accepts this 😂